services/thumbnail_queue/thumbnail_queue.py

```diff
@@ -80,7 +80,7 @@ def service(*args, **kwargs):
                 obj, err = fetch_obj(session)
                 if err is not None:
                     log(f"Error fetching object with missing thumbnails: {str(err)}")
-                    time.sleep(5)
+                    time.sleep(1)
```

Comment:
Let's go :rocket: :sweat_smile:

Reply:
These errors never occur, but if they do there's no point in sleeping really. I felt like a 1 second sleep was enough to not just absolutely destroy the logs if we do get stuck in an error, but no need for 5 seconds here.